### PR TITLE
Allow passing of additional HTTP headers in htsget.get

### PR DESCRIPTION
**Pull request by aaclan-ebi**
#40

### CODE REVIEW

**Review by jeromekelleher — COMMENTED** [review]
This looks really great, thanks @aaclan-ebi! A few tiny style suggestions, but otherwise :+1:

File: htsget/cli.py [hunk 1]
```diff
@@ -61,15 +64,21 @@ def run(args):
                 "Cannot retry failed transfers when writing to stdout. Setting "
                 "max_retries to zero")
             args.max_retries = 0
+
     exit_status = 1
+
     try:
+        headers = json.loads(args.headers) if args.headers else None
```
[the remark this answers is not in the document]
Comment (aaclan-ebi) [reply htsget/cli.py:96]: Thanks, @jeromekelleher! I've made the changes to apply your suggestion. I wanted it to exit tho when it got a header parsing error. Hope the last change is fine.

[the remark this answers is not in the document]
Comment (jeromekelleher) [reply htsget/cli.py:96]: Oh I see. Sorry, you were right and the original version was better :facepalm:

[the remark this answers is not in the document]
Comment (jeromekelleher) [reply htsget/cli.py:96]: Just snake_case and I think we're good to go!

**Comment by jeromekelleher** [comment]
Merged, thanks @aaclan-ebi!

Shall I push out a release?

**Comment by aaclan-ebi** [comment]
@jeromekelleher Yes, please! Thank you!

**Comment by jeromekelleher** [comment]
On the way.

**Comment by jeromekelleher** [comment]
Done! https://pypi.org/project/htsget/0.2.6/

**Pull request merged**